### DISCUSSION TITLE
Minio Non-anonymous request authentication and anonymous request authentication

### DISCUSSION THREAD

**Discussion by snailAlice** [minio/minio#15625] [Q&A] [answered]
What is non-anonymous request authentication and anonymous request authentication?
Whether the anonymous request authentication is to inherit the bucket policy？
Whether the non-anonymous request authentication is the union of the inherited user's policy and the bucket policy？

**Comment by klauspost** [comment, marked as answer]
MinIO follows S3 specification for policies: https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-access-control.html

https://docs.min.io/minio/baremetal/security/minio-identity-management/policy-based-access-control.html#minio-policy
